locale/ru.po

```diff
@@ -291,15 +291,11 @@ msgstr "m"
 
 #: pikaur/install_cli.py:55
 msgid "no editor found. Try setting $VISUAL or $EDITOR."
-msgstr ""
-"редактор не найден. Попробуйте установить переменные окружения $VISUAL или "
-"$EDITOR."
+msgstr "редактор не найден. Попробуйте установить переменную окружения $VISUAL или $EDITOR."
 
 #: pikaur/main.py:244
 msgid "pikaur requires systemd >= 235 (dynamic users) to be run as root."
-msgstr ""
-"pikaur требует systemd >= 235 (поддержка динамических пользователей) для "
-"запуска от root."
+msgstr "pikaur требует systemd >= 235 (поддержка динамических пользователей) для запуска от root."
```

Comment:
Well, I wanted to do these lines like this too, but I wanted to have less than 80 chars on one line

Reply:
@Yardanico I think the terminal will not have problems with the transfer of text

Reply:
doesn't really matter, i have quite strict requirements only to code